scrapers/5Kteens.yml

```diff
@@ -0,0 +1,55 @@
+name: 5Kteens
+sceneByURL:
+  - action: scrapeXPath
+    url:
+      - 5kteens.com/episodes
+      - 5kporn.com/episodes
+    scraper: sceneScraper
+xPathScrapers:
+  sceneScraper:
+    scene:
+      Studio:
+        Name:
+          selector: //span[starts-with(@class, "color-")]/@class
+          postProcess:
+            - map:
+                color-5KT: 5Kteens
+                color-5KP: 5Kporn
```

Comment:
From the draft PR the studio selector is a lot simpler
Returns studios with the spaces `5K Porn` and `5K Teens`
```yaml
      Studio:
        Name: //img[contains(@class, "top-logo")]/@alt
```

Reply:
The reason I chose the current selector is because 5K Teens scenes are also accessible via the 5K Porn domain and vice-versa, and it is tied to the scene itself.
Your suggestion is supposedly tied to the site, however I get `5K Porn` on both sites?

Reply:
I didnt notice that scenes from both where accesible in the other site.
And yes now i see what you mean. They indeed dont have the teens in all scenes.